src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,257 @@
 ---
-title: Cluster Single Sign-on (SSO) using a JSON web token (JWT)
+title: Cluster Single Sign-on (SSO) using JSON web tokens (JWTs)
 summary: Overview of Cluster Single Sign-on (SSO) for CockroachDB {{ site.data.products.core }}, review of authenticating users, configuring required cluster settings.
 toc: true
 docs_area: manage
 ---
 
-Cluster Single Sign-On (SSO) allows users to access the SQL interface of a CockroachDB cluster (whether provisioned on CockroachDB {{ site.data.products.cloud }} or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of cloud-based or customer-managed identity providers (IdPs).
+CockroachDB clusters allow users to authenticate with Single Sign-on (SSO), both to the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}), and for SQL client access, using OpenID connect (OIDC).
 
-This page describes the procedure for accessing a CockroachDB {{ site.data.products.cloud }} cluster using the JWT access tokens provided by a customer-managed IdP. This document applies for both CockroachDB {{ site.data.products.core }} and CockroachDB {{ site.data.products.dedicated }} customers.
+Cluster Single Sign-On (SSO) enables users to access the SQL interface of a CockroachDB cluster (whether provisioned on CockroachDB {{ site.data.products.cloud }} or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of cloud-based or customer-managed identity providers (IdPs). This applies for both {{ site.data.products.core }} {{ site.data.products.enterprise }} and {{ site.data.products.dedicated }} customers.
```

Comment:
Is serverless intentionally being excluded from the last sentence? It works in serverless.

Reply:
Is this better?

Reply:
Maybe I'm not following what the last sentence:`{{ site.data.products.core }} {{ site.data.products.enterprise }} and {{ site.data.products.dedicated }}` renders to. My guess was that it would exclude serverless and say something like crdb enterprise and dedicated. Am I off base?

Reply:
I think the last sentence is still missing serverless unless I'm missing